index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [],
+    this.length =this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    const list = this.items.push(item)
+    list.sort(function(a, b) {
+      return a - b
+    })
+    this.length = list.length
+    return list
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos > this.items.length-1) {
+      return 'OutOfBounds'
+    }
+    return this.items[pos]
+  }
 
-  max() {}
+  max() {
+    if (this.length = 0) {
```

Comment:
Estas igualando this.length a 0? o lo quieres comparar? si lo quieres comparar entonces tres iguales :D